agent/build.gradle

```diff
@@ -28,6 +28,13 @@ dependencies {
     }
 
     compile "net.sf.saxon:Saxon-HE:$saxonHEVersion"
+
+    //IKEA Tradfri agent dependencies
+    compile "org.eclipse.californium:californium-core:2.1.0"
```

Comment:
Any dep should have the version number placed in the `gradle.properties` file and referenced using avariable like everything else.